src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -90,6 +90,23 @@ internal sealed class VariableInformation : IVariableInformation
         private string DisplayHint { get; set; }
         public bool IsPreformatted { get; set; }
 
+        public string Address()
+        {
+            // ask GDB to evaluate "&expression"
+            string command = "&("+_strippedName+")";
+            IVariableInformation execVariable = new VariableInformation(command, this);
+            execVariable.SyncEval();
+            return _engine.DebuggedProcess.Natvis.FormatDisplayString(execVariable);
```

Comment:
>return [](start = 12, length = 6)

You probably want to verify that the result you get back looks like a pointer.

Reply:
Changed to use EvalDependentExpression() as mentioned above. `&(*test)` returns `"0x7ffcb35f89cc"` for example.